pyproject.toml

```diff
@@ -43,3 +43,27 @@ Django = "^2.2"
 [build-system]
 requires = ["poetry-core>=1.0.0a5"]
 build-backend = "poetry.core.masonry.api"
+
+
+[tool.isort]
+force_single_line = true
+force_sort_within_sections = true
+line_length = 110
+order_by_type = false
+skip_glob = [
+	"*/.tox/*",
+	"*/node_modules/*.py",
```

Comment:
is this a thing? 🙀

Reply:
I wouldn't have thought so, until I encountered it!